project/Dependencies.scala

```diff
@@ -109,6 +109,7 @@ object Dependencies {
   private val akkaSlf4j            = ("com.typesafe.akka" %% "akka-slf4j" % Versions.Akka).excludeAll(excludeSlf4j: _*)
   private val akkaStream           = "com.typesafe.akka" %% "akka-stream" % Versions.Akka
   private val akkaProtobuf         = "com.typesafe.akka" %% "akka-protobuf" % Versions.Akka
+  private val akkaProtobuf_v3      = "com.typesafe.akka" %% "akka-protobuf-v3" % Versions.Akka
```

Comment:
Why do we need to keep both versions?

Reply:
Not sure. There are some compilation errors and 'm still trying to sort out the correct steps with protobuf v3.

So far, I had to bump my local binary of `protoc` to `3.9.0`. I just made more changes on `project/Protobuf.scala` to use `3.9.0` generator, but also rename `com.google.protobuf` to `akka.protobufv3.internal` (instead of `akka.protobuf`). I'll push these changes in a bit.

See https://github.com/lagom/lagom/pull/2267/commits/48be35b67e632576c00a680edcc3c7f5ea66af75#diff-a27b5c9ac78372ce5db8508a13198d0dR65

Reply:
related https://github.com/lagom/lagom/issues/1487

Reply:
the meeting notes in https://github.com/lightbend/play-team/issues/229#issuecomment-523476499 were crucial to find https://github.com/lagom/lagom/commit/48be35b67e632576c00a680edcc3c7f5ea66af75#diff-a27b5c9ac78372ce5db8508a13198d0dR65 :-)